content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,69 @@
+# Machine Learning Model Registry
+
+A **model registry** is a central catalog for data artifacts produced in
+multiple data science projects. Different models can be discovered, audited,
+shared, tested, and deployed from here. DVC works on top of Git to enable these
+capabilities on your existing ecosystem. And other tools from
+[Iterative](https://iterative.ai/) provide even better features and usability.
+
+![](/img/ml_model_registry_placement.jpg) _A common placement for model
+registries is between development and applications. See also [Data Registries]._
+
+[data registries]: /doc/use-cases/data-registries
+
+Model catalogs give your team key collaborative capabilities:
+
+- Store, organize, and track the [versions] of different models effectively.
+- Full data provenance and lineage trails guarantee that your modeling is
+  reproducible.
+- Share and control access to any data artifact, from early-stage [experiment]
+  results to production-ready models.
+- Save performance [metrics and plots] to quickly evaluate and compare models.
```

Comment:
Save -> Store (and show?)

Reply:
Hmmm... The first bullet already reads "Store ... models effectively" -> Changed that to "Collect".
Changed this bullet to "Store and present your model artifacts" (more focused on discoverability since we mention "interface to access" in the last bullet.